resources/assets/sass/app.scss

```diff
@@ -243,7 +243,7 @@ legend {
 
 .page-meta-buttons {
     .btn {
-        float: right;
+        display: block;
```

Comment:
So originally the show talk page had the buttons stacked. When I added an archive button, it broke how the buttons stacked. It ended up looking like this
![symposium for speakers conferences](https://cloud.githubusercontent.com/assets/11747187/17185892/eedf4c9e-5400-11e6-8b8c-37e0f1b47424.png)

To fix this I made the buttons block and took off the float right. So it looks like this now
![symposium for speakers conferences](https://cloud.githubusercontent.com/assets/11747187/17185829/aa337c64-5400-11e6-861c-c11ebebb0284.png)